### ISSUE TITLE
Init project failure 

### ISSUE THREAD

**Issue by parhelium** [tmeasday/create-graphql-server#27]
Started like this:
```
➜  rootDir  npm install -g create-graphql-server
/Users/me/.nvm/versions/node/v6.9.1/bin/create-graphql-server -> /Users/me/.nvm/versions/node/v6.9.1/lib/node_modules/create-graphql-server/dist/bin/create-graphql-server.js
/Users/me/.nvm/versions/node/v6.9.1/lib
└── create-graphql-server@0.5.1
```

Error:
```
➜  rootDir create-graphql-server init generator
env: babel-node: No such file or directory
```

**Comment by jfabraxas** [comment]
i have the same problem...

**Comment by xavcz** [comment]
Have you tried installing babel-cli globally?

```npm install -g babel-cli```

**Comment by ChrisLusted** [comment]
`npm install -g babel-cli` fixed this for me. 

thanks @xavcz

**Comment by parhelium** [comment]
@xavcz That resolves the issue...
I was trying sth similar : `npm install babel-cli` - forgot to add `-g`

**Comment by tmeasday** [comment]
This sounds like a bug though, I don't think you should have to do that.

**Comment by tmeasday** [comment]
Fixed in 0.5.3
